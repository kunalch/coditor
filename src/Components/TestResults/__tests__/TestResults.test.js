import React from "react";
import renderer from "react-test-renderer";
import TestResults from "./../TestResults";

describe("<TestResults /> ", () => {
  it("should render properly", () => {
    const tree = renderer.create(
      <TestResults
        results={'{"Test-1":"Fail","Test-2":"Fail","Test-3":"Fail"}'}
      />
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
