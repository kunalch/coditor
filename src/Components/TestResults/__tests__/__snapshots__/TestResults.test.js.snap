// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestResults />  should render properly 1`] = `
<pre
  className="sucess"
>
  {"Test-1":"Fail","Test-2":"Fail","Test-3":"Fail"}
</pre>
`;
